arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/copy/ExtCodeCopySection.java

```diff
@@ -122,8 +122,10 @@ public ExtCodeCopySection(Hub hub) {
             .accountFragment()
             .makeWithTrm(accountBefore, accountAfter, rawAddress, doingDomSubStamps);
     accountDoingFragment.requiresRomlex(triggerCfi);
+    if (triggerCfi) {
+      hub.romLex().callRomLex(hub.messageFrame());
+    }
```

Comment:
We should trigger the ROM and ROM_LEX modules iff it is deemed necessary (`trigger_ROMLEX = true` or in the old notation `triggerCfi = true`)